extensions/nginx/index.js

```diff
@@ -135,13 +137,35 @@ class NginxExtension extends cli.Extension {
             skip: (ctx) => ctx.dnsfail || fs.existsSync('/etc/letsencrypt/acme.sh'),
             task: () => {
                 const acmeTmpDir = path.join(os.tmpdir(), 'acme.sh');
+                const acmeApiUrl = 'https://api.github.com/repos/Neilpang/acme.sh/releases/latest';
 
                 this.ui.logVerbose('ssl: creating /etc/letsencrypt directory', 'green');
-                // acme.sh creates the directory without global read permissions, so we need to make sure
-                // it has global read permissions first
+                // acme.sh creates the directory without global read permissions, so we need to make
+                // sure it has global read permissions first
                 return this.ui.sudo('mkdir -p /etc/letsencrypt').then(() => {
-                    this.ui.logVerbose('ssl: cloning acme.sh to temporary directory', 'green');
-                    return execa.shell(`git clone https://github.com/Neilpang/acme.sh.git ${acmeTmpDir}`);
+                    this.ui.logVerbose('ssl: downloading acme.sh to temporary directory', 'green');
+                    return fs.emptyDir(acmeTmpDir)
+                }).then(() => got(acmeApiUrl)).then((response) => {
+                    if (response.statusCode !== 200) {
+                        // @todo: Should a specific type of error be thrown?
+                        throw new Error('Unable to query GitHub for ACME download URL');
+                    }
+
+                    try {
+                        response = JSON.parse(response.body).tarball_url;
+                    } catch (E) {
+                        throw new Error('Unable to parse GitHub response');
+                    }
+                    return download(response, acmeTmpDir, {extract: true});
+                }).then(() => {
+                    // The archive contains a single folder with the structure
+                    //  `{user}-{repo}-{commit}`, but we don't know what commit is
+                    //  from the API call. Since the dir is empty (we cleared it),
+                    //  the only thing in acmeTmpDir will be the extracted zip.
+                    //  The subdir contents need to be moved up one level
+                    let subdir = fs.readdirSync(acmeTmpDir)[0];
+                    subdir = path.resolve(acmeTmpDir, subdir);
+                    return fs.move(subdir, acmeTmpDir);
```

Comment:
I'm assuming this was meant to be `/etc/letsencrypt` rather than acmeTmpDir? 😉 

Tested this out and got confused for a second lol

Reply:
I was under the impression that's [line 170-173](https://github.com/vikaspotluri123/Ghost-CLI/blob/8ce5c6276f2cda02396ce965bc345889f5a5c543/extensions/nginx/index.js#L170-L173)

Reply:
🤦‍♂️ oops, that's my bad